index.js

```diff
@@ -89,6 +89,17 @@ const schemas = {
       'account_sid',
       'application_sid'
     ]
+  },
+  system_alerts: {
+    measurement: 'system_alerts',
+    fields: {
+      status: Influx.FieldType.STRING,
```

Comment:
I think we can get rid of this in favor of having a tag 'state'.  Making it a tag value means we will be able to search on state values

Reply:
@davehorton made changes as per your recommendations, thank you